src/software/simulation/standalone_simulator_main.cpp

```diff
@@ -25,8 +25,22 @@ int main(int argc, char **argv)
                 ->setValue(args->interface()->value());
         }
 
+        // Experimentally determined restitution value
+        MutableDynamicParameters->getMutableSimulatorConfig()
+            ->mutableBallRestitution()
+            ->setValue(0.8);
+        // Measured these values from fig. 9 on page 8 of
+        // https://ssl.robocup.org/wp-content/uploads/2020/03/2020_ETDP_ZJUNlict.pdf
+        MutableDynamicParameters->getMutableSimulatorConfig()
+            ->mutableSlidingFrictionAcceleration()
+            ->setValue(6.9);
+        MutableDynamicParameters->getMutableSimulatorConfig()
+            ->mutableRollingFrictionAcceleration()
+            ->setValue(0.5);
```

Comment:
we can probably just put these in the yaml as default values?

Reply:
hm, the physicsball, physicsworld, simulator tests all depend on friction being 0

Reply:
ok fair enough, id probably reset it in the test fixture rather than here, but its more a personal preference thing so won't block on this